one-ui/src/main/java/com/marklogic/hub/oneui/auth/AuthConfig.java

```diff
@@ -17,55 +17,42 @@
 
 import com.marklogic.hub.oneui.models.EnvironmentInfo;
 import com.marklogic.hub.oneui.services.EnvironmentService;
+
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Lazy;
 import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
-import org.springframework.security.config.annotation.web.builders.WebSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
 import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
 
 /**
- * Extends Spring Boot's default web security configuration class and hooks in MarkLogic-specific classes from
- * marklogic-spring-web. Feel free to customize as needed.
+ * Extends Spring Boot's default web security configuration class and hooks in MarkLogic-specific
+ * classes from marklogic-spring-web. Feel free to customize as needed.
  */
 @Configuration
 @EnableWebSecurity
 public class AuthConfig extends WebSecurityConfigurerAdapter {
+    @Autowired private EnvironmentService environmentService;
 
-    @Autowired
-    EnvironmentService environmentService;
+    @Autowired private LoginLogoutHandler loginLogoutHandler;
 
-    private LoginLogoutHandler loginLogoutHandler = new LoginLogoutHandler();
+    @Autowired private RestAuthenticationEntryPoint restAuthenticationEntryPoint;
```

Comment:
Do you have your IDE formatter configured to put annotations on the same line as the field they refer to? I have not seen this convention before, and it seems most of the DHF code has annotations on lines above the field/method that they refer to - the "@Test" annotation being the most obvious one. Unless you have strong feelings about this, can you ensure that your IDE will keep annotations above the fields? Otherwise, we'll have a lot of changes in PRs just due to a different formatter being used, which requires reviewers to spend more time trying to understand what changed.

Reply:
I used the google style and not sure why only this file annotation was formatted with the same line. I will turn off the format and redo what I have done.

Reply:
I will add some tests too.